src/routes/committee/schedule/availability/[rehearsalId]/+page.svelte

```diff
@@ -0,0 +1,74 @@
+<script lang="ts">
+	import type { availability } from '$lib/types/domain/availability'
+	import type { rehearsal } from '$lib/types/domain/rehearsal'
+	import type { user } from '$lib/types/domain/user'
+	import {
+		Button,
+		Grid,
+		Modal,
+		StructuredList,
+		StructuredListCell,
+		StructuredListHead,
+		StructuredListRow
+	} from 'carbon-components-svelte'
+	import { Chat } from 'carbon-icons-svelte'
+
+	export let data: {
+		users: { [id: string]: user }
+		availabilities: { [id: string]: availability }
+		rehearsal: rehearsal
+	}
+	let reasonText: string = ''
+	let openReason: boolean = false
+</script>
+
+<h1>Rehearsals {data.rehearsal.startTime.toDate().toDateString()}</h1>
+<Grid>
+	<StructuredList>
+		<StructuredListHead>
+			<StructuredListRow head>
+				<StructuredListCell head>Participant</StructuredListCell>
+				<StructuredListCell head>Availability</StructuredListCell>
+			</StructuredListRow>
+		</StructuredListHead>
+		{#each Object.entries(data.users) as [id, user]}
+			<StructuredListRow>
+				<StructuredListCell>
+					{user.name}
+				</StructuredListCell>
+				<StructuredListCell>
+					{#if id in data.availabilities}
```

Comment:
```suggestion
					{#if id in data.availabilities}
					{@const availability = data.availabilities[id]}
```
maakt je leven makkelijker door de rest van de loop

Reply:
Dus vergeet niet om dan deze const te gebruiken in de rest

Reply:
oh dit mag, chill